### PR TITLE
PR Template for S3Server

### PR DESCRIPTION
**Pull request by TechWriter-Mayur**
PR Template for S3Server

Signed-off-by: TechWriter-Mayur <mayur.gupta@seagate.com>

-----
[View rendered .github/Pull_Request_Template.md](https://github.com/TechWriter-Mayur/cortx-s3server/blob/main/.github/Pull_Request_Template.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: .github/Pull_Request_Template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- _PR is self-reviewed? Yes/No?_
+- _Any interface change Yes/No?_
+- _Change notified to other gatekeepers: Yes/No?_
+- _Code reviews done and addressed: Yes/No?_
+- _Codacy issues reviewed and addressed: Yes/No?_
+- _Deployment test : Done/Not?_
+- _UT/smoke test: Done/Not?_
+- _Jira number added to PR: Yes/No?_
+- _Github merge done using UI: Yes/No?_
```
Comment (nileshgovande) [inline .github/Pull_Request_Template.md:31]: 1)side effects on other features - deployment/update/node-replacement
2)Changes to quick-start-guide/community impact

Comment (mukul-seagate11) [reply .github/Pull_Request_Template.md:31]: @TechWriter-Mayur, please add further points as suggested?

**Comment by mukul-seagate11** [comment]
@TechWriter-Mayur, also below points as suggested,

1. All dependent component code PR are raised or already merged Yes/No
2. All dependent code already merged in landing branch Yes/No

**Comment by mukul-seagate11** [comment]
@nileshgovande, can the PR be reviewed & merged

**Pull request closed without merging**